index.js

```diff
@@ -307,8 +311,12 @@ module.exports = {
     //
     // Stylistic Issues
     //
+    // enforce linebreaks after opening and before closing array brackets
+    'array-bracket-newline': 'off',
```

Comment:
Hmmm, that's probably OK for now, but I wouldn't mind revisiting this.  Although, in reality, since we use Prettier, it won't matter much.